tsaoptions.json

```diff
@@ -0,0 +1,12 @@
+{
+  "areaPath": "DevDiv\\ASP.NET Core",
```

Comment:
@Tratcher @danmoseley I bet this isn't the correct area path for Katana. What's right❔

Reply:
🤷

Reply:
DevDiv\ASP.NET OOB\Katana